packages/react-shopify-app-bridge/spec/Provider.spec.tsx

```diff
@@ -218,9 +196,10 @@ describe("GadgetProvider", () => {
         hasNext: false,
       });
 
-      expect(mockDispatch).not.toHaveBeenCalled();
       expect(container.outerHTML).toMatchInlineSnapshot(`"<div><span>hello world</span></div>"`);
       expect(mockNavigate).not.toHaveBeenCalled();
+
+      window.shopify.environment.embedded = false;
     });
```

Comment:
Do you want to add any tests / error messages around the shopify global being absent?

Reply:
will take a look at this